api/services/tasks/v1/tasks.proto

```diff
@@ -46,6 +47,8 @@ service Tasks {
 	rpc Checkpoint(CheckpointTaskRequest) returns (CheckpointTaskResponse);
 
 	rpc Update(UpdateTaskRequest) returns (google.protobuf.Empty);
+
+	rpc Metrics(types.MetricsRequest) returns (types.MetricsResponse);
```

Comment:
Types should not include request/response messages. Just keep them with their service.